lectures/solow.md

```diff
@@ -55,9 +55,11 @@ $$
 Production functions with this property include
 
 * the **Cobb-Douglas** function $F(K, L) = A K^{\alpha}
-  L^{1-\alpha}$ with $0 \leq \alpha \leq 1$ and
+  L^{1-\alpha}$ with $0 \leq \alpha \leq 1$. Here, $\alpha$ is the output elasticity of capital.
```

Comment:
@SylviaZhaooo we will need to move this second sentence to below the dot points and combine with the other clarifying sentence.